polyfill.js

```diff
@@ -1,9 +1,7 @@
 // Array.prototype.findIndex - MIT License (c) 2013 Paul Miller <http://paulmillr.com>
 // For all details and docs: <https://github.com/paulmillr/Array.prototype.findIndex>
 (function (globals) {
-  if (Array.prototype.findIndex) return;
-
-  var findIndex = function(predicate) {
+  Array.prototype.findIndex = function findIndex (predicate) {
```

Comment:
it'd be nice to change this file so the implementation could be used without the shimming behavior - ie, move the shim assignment into `detect.js`?